service/frontend/handler.go

```diff
@@ -1697,3 +1723,18 @@ func createServiceBusyError() *gen.ServiceBusyError {
 	err.Message = "Too many outstanding requests to the cadence service"
 	return err
 }
+
+func (wh *WorkflowHandler) getWorkflowTypeTaskList(domainID string, execution *gen.WorkflowExecution) (*gen.WorkflowType, *gen.TaskList, error) {
```

Comment:
this can be eliminated.

Reply:
why?
this is a helper function for workflow type and its corresponding task list